node-installer/node-installer.go

```diff
@@ -199,19 +199,23 @@ func patchContainerdConfig(runtimeName, basePath, configPath string, platform pl
 		existing = constants.ContainerdBaseConfig()
 	}
 
+	snapshotterName := "no-snapshotter"
```

Comment:
Why is this not `tardev-$hash`?

Reply:
It is set to that a few lines below in case we are on a platform that uses a snapshotter.

Reply:
Ah, I see - thanks.

Reply:
Interestingly, we're already switching on the platform outside these functions: https://github.com/edgelesssys/contrast/blob/59826da4deb184be42231a0b0bbf52634080a8db/node-installer/node-installer.go#L128-L141.

Reply:
That's indeed confusing (but necessary with the current structure, as some platforms won't call patchContainerdConfig.